cmd/migrate/executor/cstor_pool.go

```diff
@@ -87,6 +87,8 @@ func (m *MigrateOptions) RunCStorSPCMigrate() error {
 		return errors.Errorf("Failed to migrate cStor SPC : %s", m.spcName)
 	}
 	klog.Infof("Successfully migrated spc %s to cspc", m.spcName)
-
+	klog.Infof("Make sure to migrate the associated PVCs, " +
+		"to find the old PVCs use `kubectl get cstorvolume.openebs.io -n <openebs-namespace>`, " +
+		"and to get the migrated PVCs use `kubectl get cstorvolume.cstor.openebs.io -n <openebs-namespace>`")
```

Comment:
```suggestion
		"and to list CStorVolume for the migrated PVs use `kubectl get cstorvolume -n %s -l=openebs.io/cstor-pool-cluster=%s`", m.openebsNamespace, m.spcName, m.openebsNamespace, m.cspcName)
```
So the command can be copy pasted.